content/tutorial/02-advanced-orders/01-eth-flow/01-create-eth-flow/README.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Eth-Flow
+---
+
+There are two types of wallets:
+ - EOA (externally owned account) wallets, which are controlled by a private key
+ - Smart contract wallets, which are controlled by a smart contract
+
+Since smart contract wallets are controlled by a smart contract, they can't sign transactions using `EIP-712`.
+Anyway, CoW Protocol supports smart contract wallets by allowing them to create pre-signed orders.
+The key of pre-signed orders is a transaction to Settlement contract that proves the ownership of an order.
+
+In this example we will create a pre-signed order using Safe, but you can use any smart contract wallet.
+
+## Required dependencies
+
+For pre-signed orders, we need to use:
+ - `OrderBookApi` to send an order to the CoW Protocol order-book
+ - `MetadataApi` to generate order meta data
+ - `Safe` to create and sign the transaction to the Settlement contract
+ - `SafeApiKit` to propose the transaction to Safe owners
+
+## Define the order parameters
+
+First of all, we should define the order parameters.
+The description of each parameter can be found in the [Order structure docs.](https://TODO)
+
+```
+const defaultOrder: UnsignedOrder = {
+  receiver: safeAddress,
+  buyAmount: '650942340000000000000',
+  buyToken: '0x91056D4A53E1faa1A84306D4deAEc71085394bC8',
+  sellAmount: '100000000000000000',
+  sellToken: '0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6',
+  validTo: Math.round((Date.now() + 900_000) / 1000),
+  appData: '0x',
+  feeAmount: '0',
+  kind: OrderKind.SELL,
+  partiallyFillable: true,
+  +++signingScheme: SigningScheme.PRESIGN+++
+}
+```
+
+> `signingScheme` is the only difference between a regular order and a pre-signed order
+
+## Order meta data
+
+For analytics purposes, we can add some metadata to the order.
+The metadata is a JSON document that is stored in IPFS and referenced by the order.
+The metadata is optional, but it's recommended to add it.
+After order creation, the metadata will be displayed in [the Explorer](https://explorer.cow.fi/).
```

Comment:
Should we link to the app-data tutorial here?

Reply:
Yep, will do that to de-dupe on the advanced orders!